apps/tests/src/e2e/api-call.test.ts

```diff
@@ -0,0 +1,8 @@
+import { expect, test } from "@playwright/test";
+
+test.describe("api calls", () => {
+  test("should return text/plain", async () => {
+    const response = await fetch("http://localhost:3000/api/text-plain");
+    expect(await response.text()).toBe("test");
```

Comment:
This test is failing for me. I think we need to change this file `apps/tests/src/routes/api/text-plain.tsx` to use the following instead of `e.nativeEvent.respondWith`

```
export function GET() {
  return new Response("test");
}
```